go/minterm/minterm.go

```diff
@@ -37,6 +40,7 @@ func New() (*MinTerm, error) {
 
 // Shutdown closes the terminal.
 func (m *MinTerm) Shutdown() error {
+	m.restore()
```

Comment:
No. Restore checks the raw flag first. 

> On Jun 22, 2016, at 4:30 PM, Maxwell Krohn <notifications@github.com> wrote:
> 
> In go/minterm/minterm.go:
> 
> > @@ -37,6 +40,7 @@ func New() (*MinTerm, error) {
> >  
> >  // Shutdown closes the terminal.
> >  func (m *MinTerm) Shutdown() error {
> > +	m.restore()
> so we can double restore in some cases?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>